pkg/controller/operators/olm/operator.go

```diff
@@ -349,6 +398,22 @@ func (a *Operator) namespaceAddedOrRemoved(obj interface{}) {
 			}
 		}
 	}
+	// find any API services owned by this namespace and requeue them
+	stringSelector := fmt.Sprintf("%v==%v", ownerutil.OwnerNamespaceKey, namespace.GetName())
+	labelSelector, err := labels.Parse(stringSelector)
```

Comment:
this seems like a useful thing to factor out into ownerutil, no?

Reply:
Well, I'm not sure if the code is even needed. I'm specifically testing for when OLM is being terminated via namespace deletion, which it doesn't look like will allow/guarantee this code to run. Instead, the api service sync loop upon starting up will detect orphaned api services and get rid of them.